sync_settings/libs/gist.py

```diff
@@ -63,14 +65,14 @@ def make_uri(endpoint=''):
     def create(self, data):
         if not isinstance(data, dict) or not len(data):
             raise ValueError('Gist can`t be created without data')
-        return self.__do_request('post', self.make_uri(), data=sublime.encode_value(data, True)).json()
+        return self.__do_request('post', self.make_uri(), data=sublime.encode_value(data, True).encode("utf-8")).json()
 
     @auth
     @with_gid
     def update(self, gid, data):
         if not isinstance(data, dict) or not len(data):
             raise ValueError('Gist can`t be updated without data')
-        return self.__do_request('patch', self.make_uri(gid), data=sublime.encode_value(data, True)).json()
+        return self.__do_request('patch', self.make_uri(gid), data=sublime.encode_value(data, True).encode("utf-8")).json()
```

Comment:
can you add a test replicating this escenario?

Reply:
@mfuentesg what kind of test do you want? Just ensure that non utf-8 characters are actually encoded?

Reply:
Usually i add tests to ensure that i am fixing the found issue.
In this case, we need to use an "invalid" file content, and the plugin must be able to encoding it.

Reply:
Gotcha, I will do that now